### ISSUE TITLE
Add possibility to add a new problem from the UI

### ISSUE THREAD

**Issue by oskarhinc** [benetech/MathShare#370]
Follow the mockups from #364 and add a possibility to add a new problem to the problem set. This issue should be done after #369, to avoid problems with the compatibility of those ones. After a user clicks on the problem card, he should be routed to the problem creation page and after the problem set is saved it should be saved too.

**Comment by kkrawczyk123** [comment]
The possibility of adding new problems has been added and it works. I have created the new issue with my feedback and improvement suggestions in https://github.com/benetech/MathShare/issues/388.
